src/components/modeler/Selection.vue

```diff
@@ -597,11 +602,45 @@ export default {
             properties: {
               clientX: shape.model.get('position').x,
               clientY: shape.model.get('position').y,
+              attachedToRefId: shape.model.component.node.definition.get('attachedToRef')?.id ?? null,
             },
           });
+          const boundariesChanges = this.getBoundariesChangesForShape(shape);
+          changed = changed.concat(boundariesChanges);
         });
       return changed;
     },
+
+    /**
+     * Get properties for each boundary inside a shape
+     */
+    getBoundariesChangesForShape(shape) {
+      let boundariesChanged = [];
+      const boundaryEventTypes = [
+        boundaryErrorEventId,
+        boundaryConditionalEventId,
+        boundaryEscalationEventId,
+        boundaryMessageEventId,
+        boundarySignalEventId,
+        boundaryTimerEventId,
+      ];
+
+      const boundaryNodes = window.ProcessMaker.$modeler.nodes.filter(node => boundaryEventTypes.includes(node.type));
```

Comment:
Uhmm I'm pretty sure this nodes are available in the store?
Also, `window.ProcessMaker.$modeler` is a ref I believe also. Can you confirm @rodriquelca